kong/templates/nginx.lua

```diff
@@ -26,6 +26,10 @@ wasm {
 > for _, module in ipairs(wasm_modules_parsed) do
   module $(module.name) $(module.path);
 > end
+# injected nginx_wasm_* directives
+> for _, el in ipairs(nginx_wasm_directives) do
+  $(el.name) $(el.value);
+> end
```

Comment:
I personally would be a bit more comfortable if we guard all of the wasm directives using 'conf.wasm' as well (in addition to wasm_modules_parsed). But that is me being conservative. I understand that the end result of the current code and what I'm suggesting will be the same so it is up to you.
Same comment for nginx_kong.lua.

Reply:
Yes, I also prefer to lean towards defensive coding. Changed them.